scripts/gis-material-update/automation/automation.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+for tormays_source in $*
+do
+    # Fetch data
+    echo "Fetching $tormays_source ..."
+    case $tormays_source in
+    hsl)
+        sh /haitaton-gis/fetch_all.sh $tormays_source hki
+        ;;
+    tram_infra)
+        sh /haitaton-gis/fetch_all.sh hki osm helsinki_osm_lines
+        ;;
+    tram_lines)
+        sh /haitaton-gis/fetch_all.sh hki hsl
+        ;;
+    liikennevaylat)
+        sh /haitaton-gis/fetch_all.sh $tormays_source central_business_area ylre_katuosat
+        ;;
+    *)
+        sh /haitaton-gis/fetch_all.sh $tormays_source
+        ;;
+    esac
+
```

Comment:
Maybe the script should check the exit code of the previous step and only continue if it was 0?

Reply:
Added error handling